components/movementsensor/replay/replay_utils_test.go

```diff
@@ -225,116 +228,116 @@ func resourcesFromDeps(t *testing.T, r robot.Robot, deps []string) resource.Depe
 }
 
 // createDataByMovementSensorMethod will create the mocked structpb.Struct containing the next data returned by calls in tabular data.
-func createDataByMovementSensorMethod(method string, index int) *structpb.Struct {
```

Comment:
A lot of the seemingly big changes below are due to changing the order of the methods to match the order of of the methods in `Properties`

Reply:
What is the arguement for using the order of methods in Properties as opposed to the order the endpoints are defined in in `movementsensor.go` and implemented in in `replay.go`?

Reply:
I noticed as well that the order of the methods in `Properties` is different from the order of the endpoints. My motivation came mainly from working on the `Properties`, and because the order of the methods is all shuffled in various places, I simply decided to pick the order in `Properties` - no real motivation behind it.

I prefer for all the methods to follow the same order, so I think it's best for me to change them to follow the order of the endpoints. In the end, this helps make reading the code easier!

Reply:
Changed the order to match the endpoints. Did so additionally in a few other files.